pkg/kudoctl/cmd/init/prereqs.go

```diff
@@ -155,12 +169,18 @@ func PrereqManifests(opts Options) ([]string, error) {
 
 // Prereq returns the slice of prerequisite objects for KUDO
 func Prereq(opts Options) []runtime.Object {
-	ns := namespace(opts.Namespace)
-	svc := serviceAccount(opts)
-	rbac := roleBinding(opts)
-	secret := webhookSecret(opts)
+	var prereqs []runtime.Object
+
+	if opts.Namespace == defaultns {
```

Comment:
I don't like this duplication (and it's not your fault!) 

btw. If you want to follow up with another PR that makes Prereq be used in `installPrereq` that would be a good cleanup :) If not, I'll do that when this one merges

Reply:
Sounds good, if you can provide me some tips: I'm not sure how to do this because `installPrereq` has only `kubernetes.Interface` client which operates on typed kubernetes objects, and `Prereq` returns a slice of untyped objects, which (as I saw in tests) should be managed by [controller-runtime Client](https://github.com/kubernetes-sigs/controller-runtime/blob/v0.2.0/pkg/client/interfaces.go#L104).